applications/nrf5340_audio/src/modules/hw_codec.c

```diff
@@ -316,3 +319,102 @@ int hw_codec_init(void)
 
 	return 0;
 }
+
+static int cmd_input(const struct shell *shell, size_t argc, char **argv)
+{
+	int ret;
+	uint8_t idx;
+
+	enum hw_codec_input {
+		LINE_IN,
+		PDM_MIC,
+		NUM_INPUTS,
+	};
+
+	if (argc != 2) {
+		shell_error(shell, "Wrong number of arguments provided %d", argc);
```

Comment:
I still think we can be specific and say something like: "Only one argument required, not %d"